docs/upgrading-to-v5.md

```diff
@@ -0,0 +1,37 @@
+# Upgrading to v5
+
+This document lists the full set of changes between version 4 and version 5 that are relevant to anyone wanting to upgrade to the new version. These update documents groups changes into "breaking changes", "new features", and "deprecations" across both the "standard" and "attribution" builds (see [build options](/#build-options) for details).
+
+## ❌ Breaking changes
+
+### Standard build
+
+#### General
+
+- **Removed** First Input Delay (FID) support and `onFID()` [as deprecated in v4](./upgrading-to-v4.md#%EF%B8%8F-deprecations). FID has been replaced by INP and this [removal was previously advertised in that announcement post](https://web.dev/blog/inp-cwv-launch#fid_deprecation_timeline). ([#519](https://github.com/GoogleChrome/web-vitals/pull/519))
+- **Changed** the browser support policy to [Baseline Widely available](https://web.dev/baseline) browser support. ([#525](https://github.com/GoogleChrome/web-vitals/pull/525))
+
+##### More details on the Baseline Widely available change:
+
+The pre-built [packages of the `web-vitals` library available on CDNs](/README#load-web-vitals-from-a-cdn) are built with this Baseline Widely available support in mind. This should not be an issue for modern browsers released in the last few years, which are usually required in order to measure the metrics this library is designed for.
```

Comment:
```suggestion
All of the [builds](README#build-options) in `web-vitals` v5 use only [Baseline Widely available](https://web.dev/baseline) APIs, which means they should run without error in all browsers released in the last few years. Note that the Core Web Vitals metrics are only available in modern browsers, so legacy browser support is unnecessary for this library.

If your site needs to support legacy browsers, you can still use the `web-vitals` library without causing errors in those browsers by adhering to the following recommendations:
```

Reply:
I think we should call out that packages on unpkg and the like are affected more and don't think your suggestions do that strongly enough. WDYT?

Reply:
Why are they affected more?

Reply:
Yeah I guess I'm reading "build options" as more of a way of building it, rather than "pre-built packages available to install from CDNs". But guess it's the same.

And, to your other point, these pre-built options WILL at least be in their own `<script>` (and hopefully `<script type="module">`!) elements.

Let's go with your suggestion then.